templates/desktop/features.html

```diff
@@ -1,384 +1,586 @@
-          <p class="p-matrix__desc">A fast and secure desktop messaging app, perfectly synced with your mobile phone.</p>
+  <div>
+    <div class="row--25-75">
+      <div class="col">
+        <hr class="u-hide--small" />
+        <div class="row">
+          <div class="col-3 col-medium-3 col-small-4">
+            <div class="row">
+              <div class="col-6 col-medium-6 col-small-1">
+                <div class="p-image-wrapper">
+                  {{
+                  image(
+                  url="https://assets.ubuntu.com/v1/9917aba6-spotify-linux-256.png",
+                  alt="Spotify icon",
+                  height="32",
+                  width="32",
+                  hi_def=True,
+                  ) | safe
+                  }}
+                </div>
+              </div>
+              <div class="col-6 col-medium-6 col-small-3">
+                <h3 class="p-heading--4 u-sv-2"><a href="https://snapcraft.io/spotify">Spotify</a></h3>
+                <p>Play and stream your favourite songs, playlists and albums for free with Spotify.</p>
+              </div>
+            </div>
+          </div>
+          <hr class="u-hide--medium u-hide--large" />
+          <div class="col-3 col-medium-3 col-small-4">
+            <div class="row">
+              <div class="col-6 col-medium-6  col-small-1">
+                <div class="p-image-wrapper">
+                  {{
+                  image(
+                  url="https://assets.ubuntu.com/v1/4bb576dc-bitwarden.svg",
+                  alt="Bitwarden icon",
+                  height="32",
+                  width="32",
+                  hi_def=True,
+                  attrs={"class": "u-vertically-center"},
+                  ) | safe
+                  }}
+                </div>
+              </div>
+              <div class="col-6 col-medium-6 col-small-3">
+                <h3 class="p-heading--4 u-sv-2"><a href="https://snapcraft.io/bitwarden">Bitwarden</a></h3>
+                <p>A secure and free password manager for all of your devices.</p>
+              </div>
+            </div>
+          </div>
+          <hr class="u-hide--large" />
+          <div class="col-3 col-medium-3 col-small-4">
+            <div class="row">
+              <div class="col-8 col-medium-6 col-small-1">
+                <div class="p-image-wrapper">
+                  {{
+                  image(
+                  url="https://assets.ubuntu.com/v1/7b460167-vlc-icon.png",
+                  alt="VLC icon",
+                  height="32",
+                  width="32",
+                  hi_def=True,
+                  attrs={"class": "u-vertically-center"},
+                  ) | safe
+                  }}
+                </div>
+              </div>
+              <div class="col-8 col-medium-6 col-small-3">
+                <h3 class="p-heading--4 u-sv-2"><a href="https://snapcraft.io/vlc">VLC player</a></h3>
```

Comment:
```suggestion
                <h3 class="p-heading--4 u-sv-2"><a href="https://snapcraft.io/vlc">VLC</a></h3>
```
Just VLC on the doc

Reply:
Iirc I was told by @juanruitina that the text and most of the images (if not specified otherwise in Figma design) should stay the same. And this one is an old text from how it was before. Juan, please let me know if I need to change this.